ui/00_message/ui/clan_kyoten.xml

```diff
@@ -54,6 +56,9 @@
     <original>以下を購入します。よろしいですか？
 　拡張項目：&lt;VAL&gt;
 　購入価格：&lt;VAL CP&gt;</original>
+    <translation>Purchasing the following. Is this Ok?
```

Comment:
Most (if not all) prompts that involve the "Is this OK?" style have been localized more like "Are you sure you want to purchase the following?" because it reads better.